src/signals/settings/categories/Detail/components/CategoryForm/CategoryForm.tsx

```diff
@@ -95,6 +97,27 @@ const CategoryForm: FunctionComponent<CategoryFormProps> = ({
     [data]
   )
 
+  useEffect(() => {
+    if (data?.is_public_accessible) {
+      setIsPublicAccessible(data.is_public_accessible)
+    }
```

Comment:
I think you can skip this bit when line 88 is this:
```javascript
const [isPublicAccessible, setIsPublicAccessible] = useState(data.is_public_accessible ?? false)
```

Reply:
done